### ISSUE TITLE
Use of :FINAL in MM patches

### ISSUE THREAD

**Issue by RedAV8R** [sumghai/SDHI_ServiceModuleSystem#118]
Not sure if it's intended or not, but the use of :FINAL in your MM patches is hard for others to modify things. I do understand the want to prevent others from doing so and the troubleshooting involved. If I submit a PR to replace them would you be receptive to that?

The TACLS patch is outdated as well.

**Comment by sumghai** [comment]
> Not sure if it's intended or not, but the use of :FINAL in your MM patches is hard for others to modify things. I do understand the want to prevent others from doing so and the troubleshooting involved. If I submit a PR to replace them would you be receptive to that?

Some of the instances are deliberately meant to be final (e.g. the patch that adds the umbilical model to the pod, the life support patches).

I can definitely consider PRs on a case-by-case basis, but you'll need to justify each and every modification.

> The TACLS patch is outdated as well.

Has the current maintainer changed things around yet again?

Might be good to post this as a separate issue.

**Comment by sumghai** [comment]
@RedAV8R, I am waiting on your pull requests.

**Comment by RedAV8R** [comment]
#119 There you go. Sorry about the wait.

**Comment by sumghai** [comment]
As per discussions in #119, PR accepted and merged with thanks!
